app/api/models/admin.py

```diff
@@ -11,3 +11,43 @@ def add_models_to_namespace(api_namespace):
         description='The unique identifier of a user'
     )
 })
+
+
+public_admin_user_api_model = Model('User list model', {
```

Comment:
shouldn't you do the same for this model, as in line 5 of this file?

Reply:
Sure. fixed it

Reply:
Sure. Fixed it